libs/utils/src/bin_ser.rs

```diff
@@ -408,4 +413,44 @@ mod tests {
         let msg2 = LongMsg::des(&encoded).unwrap();
         assert_eq!(msg, msg2);
     }
+
+    #[test]
```

Comment:
A comment saying it checks that struct wrapper doesn't change the serialization would be nice.